### PR TITLE
Webdev3

### PR DESCRIPTION
**Pull request by javimacaya-umd**

### CODE REVIEW

**Comment by kristofj-umd** [comment]
Oh ❗ The amount of commits taking place here.  _increíble_

Great work on this assignment.

File: webdev3.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            color: blueviolet;
+        }
+        
+        /* put this text in line with other text in header */
+        #header h5 {
+            margin-top: 25px;
+        }
+        
+        /* Latitude/Longitude coordinates element styling */
+        /* Adapted from source 1: https://docs.mapbox.com/mapbox-gl-js/example/mouse-position/ and adapted from source 2:view-source:https://kristofj-umd.github.io/webdev-f19/assignments/webdev3.html */
```
Comment (kristofj-umd) [inline webdev3.html:83]: Excellent source citation

File: webdev3.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            Umd.bindPopup("<h2>UMD</h2> <p>University of Minnesota Duluth</p>");
+        </script>
+    
+    <! -- Displaying latitude/longitude coordinates as mouse cursor moves around map -->
+    <! -- Adapted from source 1: https://docs.mapbox.com/mapbox-gl-js/example/mouse-position/ and adapted from source 2:view-source:https://kristofj-umd.github.io/webdev-f19/assignments/webdev3.html--> 
+        <script>
+            map.on('mousemove', function(cursorMoving) {
+                document.getElementById('mapCoordinates').innerHTML = "Current position = " +JSON.stringify(cursorMoving.latlng);
+            }); 
+        </script>
```
Comment (kristofj-umd) [inline webdev3.html:166]: Great job getting this to work!

**Review by kristofj-umd — COMMENTED** [review]
Excellent work here

File: webdev3.html (hunk 1 above)
Comment (javimacaya-umd) [reply webdev3.html:83]: Thanks Kris!

File: webdev3.html (hunk 2 above)
Comment (javimacaya-umd) [reply webdev3.html:166]: Thanks Kris!

Comment (javimacaya-umd) [reply webdev3.html:166]: I really appreciate your comments

**Pull request merged**